packages/remix-dev/vite/plugin.ts

```diff
@@ -26,7 +26,7 @@ import {
   type RemixConfig as ResolvedRemixEsbuildConfig,
   resolveConfig as resolveRemixEsbuildConfig,
 } from "../config";
-import { type Manifest as BrowserManifest } from "../manifest";
+import { type Manifest as RemixManifest } from "../manifest";
```

Comment:
This type has been renamed for clarity since the manifest type is shared between client and server, and especially now that its contents can differ too.